NiL.JS/BaseLibrary/Math.cs

```diff
@@ -153,6 +226,39 @@ public static JSValue exp(JSValue value)
             return res;
         }
 
+        [DoNotEnumerate]
+        [DoNotDelete]
+        public static JSValue expm1(JSValue value)
+        {
+            var x = Tools.JSObjectToDouble(value);
+            var res = 0.0;
+            if (x >= -1.0 && x <= 1.0) // for better accuracy
+            {
+                double f = 1.0;
+                double _x = x;
+                double _res = x;
+                int i = 2;
+                while (res != _res)
+                {
+                    res = _res;
+                    _x *= x;
+                    f *= i++;
+                    _res += _x / f;
```

Comment:
Do not use '_' in names of local variables. This character is used for private fields.